io.github.uowuo.abaddon.yml

```diff
@@ -0,0 +1,158 @@
+app-id: io.github.uowuo.abaddon
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: abaddon
+finish-args:
+  # Wayland access
+  - --socket=wayland
+  # X11 fallback
+  - --socket=fallback-x11
+  - --share=ipc #x11 depends on this
+  # Needs network
+  - --share=network
+  # Needs sound + mic
+  - --socket=pulseaudio
+  - --device=all
+  - --env=LD_LIBRARY_PATH=/app/lib/
+  - --env=LD_LIBRARY_PATH=/app/lib/lib64/
+  # Prevents freeze when closing while connected.
+  #- --socket=session-bus
+  - --talk-name=org.gtk.vfs.*
+  - --own-name=com.github.uowuo.abaddon
+
+modules:
+  - name: abaddon
+    buildsystem: simple
+    build-commands:
+
+      - install -D abaddon.sh /app/bin/abaddon
+      - install -Dm6444 io.github.uowuo.abaddon.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 io.github.uowuo.abaddon.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm644 icon.svg /app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      #- mkdir /app/abaddon
+      - cp -r * /app/
+      - chmod +x /app/abaddon
+    sources:
+      - type: archive
+        only-archives:
+          - x86_64
+          #nightly has commits only a few times per month
+        url: https://nightly.link/uowuo/abaddon/workflows/ci/master/build-linux-MinSizeRel.zip
```

Comment:
why not building from source?

Reply:
also this is not versioned so this will randomly break. Pull from a specific commit / build. Flathub is NOT for continuous integration.